slack_sdk/web/async_client.py

```diff
@@ -5163,15 +5163,16 @@ async def users_profile_set(
     async def views_open(
         self,
         *,
-        trigger_id: str,
+        trigger_id: Optional[str] = None,
```

Comment:
The only way to accept both in Python is to make both optional..

Reply:
We could add some logic that raises a TypeError if both fields are `None` but this may not be needed if the API error mentions a missing `trigger_id` or `interactivity_pointer` field